messages/create.sandbox.md

```diff
@@ -123,3 +123,19 @@ The poll interval (%d seconds) can't be larger that wait (%d in seconds)
 # error.noCloneSource
 
 Could not find the clone sandbox name "%s" in the target org.
+
+# error.userGroupQueryFailed
+
+Could not query user group.
+
+# error.bothApexClassIdAndNameProvided
+
+Could not specify both apexClassId and apexClassName.
+
+# error.bothUserGroupIdAndNameProvided
+
+Could not specify both userGroupId and userClassName.
+
+# error.apexClassQueryFailed
+
+Could not query apex class.
```

Comment:
This error might be confusing to users (they didn't wrote the definition file, just trying to create a sandbox). Which class the msg refers to? where is this class defined?

Maybe something like this?
```
Unable to find the ID of Apex class <className> defined in the definition file.
```

Reply:
Slight edit to Cristian's suggestion:

`Unable to find the ID of the Apex class <className> that's defined in the definition file.`